lib/arux_app/api/auth.rb

```diff
@@ -41,6 +41,8 @@ def initialize(options = {})
         raise API::InitializerError.new(:client_id, "can't be blank") if self.client_id.to_s.empty?
         raise API::InitializerError.new(:client_secret, "can't be blank") if self.client_secret.to_s.empty?
         raise API::InitializerError.new(:redirect_uri, "can't be blank") if self.redirect_uri.to_s.empty?
+
+        self.api_key = options[:api_key]
```

Comment:
Can this go above the error handling and right underneath 
`self.current_user_uuid = options[:current_user_uuid]`